ksqldb-functional-tests/src/test/resources/query-validation-tests/group-by.json

```diff
@@ -2921,6 +2921,90 @@
         {"topic": "OUTPUT", "key": {"F1": 1}, "value": {"SUM": 1}},
         {"topic": "OUTPUT", "key": {"F1": 1}, "value": {"SUM": 2}}
       ]
+    },
+    {
+      "name": "stream-table with PROTOBUF schema and schema_id",
```

Comment:
This name makes it sound like the test is for a join. How about simply "PROTOBUF schema with schema_id" instead? And similar for the other new GROUP BY test.